CRSFforArduino.cpp

```diff
@@ -254,6 +288,43 @@ uint16_t CRSFforArduino::rcToUs(uint16_t rc)
     return (uint16_t)((rc * 0.62477120195241F) + 881);
 }
 
+#if (CRSF_USE_TELEMETRY > 0)
+#if (CRSF_TELEMETRY_DEVICE_ATTITUDE > 0)
+/**
+ * @brief Writes the attitude telemetry data to the CRSF frame.
+ *
+ * @param roll Roll rotation in radians.
+ * @param pitch Pitch rotation in radians.
+ * @param yaw Yaw rotation in radians.
+ */
+void CRSFforArduino::writeAttitudeTelemetry(float roll, float pitch, float yaw)
+{
+    _crsfAttitude.roll = roll * DEG_TO_RAD;
+    _crsfAttitude.pitch = -pitch * DEG_TO_RAD;
+    _crsfAttitude.yaw = yaw * DEG_TO_RAD;
```

Comment:
This is where I am currently having issues with attitude data feeding the handset's artificial horizon inaccurate data.

I am testing my code with my RadioMaster TX16S Max Edition & I am using [iNav's OpenTX Telemetry Widget](https://github.com/iNavFlight/OpenTX-Telemetry-Widget).

Currently, I am only focusing on tuning the pitch axis & getting that to display accurate information on the artificial horizon indicator on my handset, as the other two axes will fall into place, once I have the correct tuning for the pitch axis.

The irony here is... the data coming straight off of the IMU is stupidly accurate & is in Euler angles (Roll, Pitch & Yaw). Here, I am converting that to radians & (by rights) _should_ be accurate on my handset.
However, the artificial horizon is way off in the weeds relative to what the Euler angles that I am seeing, here.